src/components/Button.js

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+
+const THEME = {
+  DARK: "dark",
+  LIGHT: "light",
+};
+
+/*
+const Button = (props) => {
+    // const theme = props.theme || THEME.DARK
+    return <ButtonWrapper theme={theme}/>
+}
+ */
+
+const Button = styled.button`
```

Comment:
since you don't have a themed website, this is a nice example on how to pass props into styled components, so let's keep this as it is for future reference but it will make more sense **for this project** to have a Primary Button and a Secondary Button.